packages/web/src/components/usdc-manual-transfer/USDCManualTransfer.tsx

```diff
@@ -30,12 +43,35 @@ const messages = {
   learnMore: 'Learn More',
   copy: 'Copy Wallet Address',
   goBack: 'Go Back',
-  copied: 'Copied to Clipboard!'
+  copied: 'Copied to Clipboard!',
+  buy: (amount: string) => `Buy $${amount}`
 }
 
-export const USDCManualTransfer = ({ onClose }: { onClose: () => void }) => {
+export const USDCManualTransfer = ({
+  onClose,
+  source,
+  amountInCents,
+  onSuccessAction
+}: {
+  onClose: () => void
+  source: 'add-funds' | 'purchase'
```

Comment:
nit: if this is only controlling visibility of the price, maybe presence of `amountInCents` is enough and we don't need it?